src/analyze/analyze.c

```diff
@@ -509,9 +538,13 @@ static int pretty_boot_time(sd_bus *bus, char **_buf) {
         if (t->initrd_time > 0)
                 size = strpcpyf(&ptr, size, "%s (initrd) + ", format_timespan(ts, sizeof(ts), t->userspace_time - t->initrd_time, USEC_PER_MSEC));
 
-        size = strpcpyf(&ptr, size, "%s (userspace) ", format_timespan(ts, sizeof(ts), t->finish_time - t->userspace_time, USEC_PER_MSEC));
+        size = strpcpyf(&ptr, size, "%s (end of boot) ", format_timespan(ts, sizeof(ts), t->finish_time - t->userspace_time, USEC_PER_MSEC));
```

Comment:
So... now this is unclear. If you kept "userspace" here, then "multi-user.target reached after xxxs in userspace" is clear. But when this is "end of boot", it's not clear that the next line is counted from here. What about changing this phrase back? I think "Startup finished" is good enough to tell people that the first line is about boot finishing.